.github/workflows/build.yml

```diff
@@ -71,46 +71,3 @@ jobs:
 
       - name: Install
         run: make install
-
-  with-external-deps:
-    runs-on: ubuntu-22.04
-    timeout-minutes: 10
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Install dependencies
-        run: |
-          sudo add-apt-repository ppa:neovim-ppa/stable
-          ./.github/scripts/install_deps.sh
-          sudo apt-get install -y \
-            libluajit-5.1-dev \
-            libmsgpack-dev \
-            libtermkey-dev \
-            libtree-sitter-dev \
-            libunibilium-dev \
-            libuv1-dev \
-            lua-busted \
-            lua-filesystem \
```

Comment:
Technically we still use `require('lfs')` in several places, but I think we accidentally get it from e.g. busted.

I think we could eliminate all those `require('lfs')` eventually, and use `vim.fs` instead.

Reply:
I see. Should I re-add `lua-filesystem`, or did you mean this as a reference for later?

Reply:
I've re-added it.